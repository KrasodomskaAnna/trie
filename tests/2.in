+ v p
+ a m
+ o w
+ f l
+ e x
+ d h
+ u c
+ r k
+ s f
+ x t
? a
? b
+ b u
? c
+ c y
? d
? e
? f
? g
+ g r
? h
+ h n
? i
+ i o
? j
+ j v
? k
+ k a
? l
+ l i
? m
+ m z
? n
+ n q
? o
? p
+ p s
? q
+ q g
? r
? s
? t
+ t j
? u
? v
? w
+ w e
? x
? y
+ y d
? z
+ z b
? a
? b
? c
? d
? e
? f
? g
? h
? i
? j
? k
? l
? m
? n
? o
? p
? q
? r
? s
? t
? u
? v
? w
? x
? y
? z