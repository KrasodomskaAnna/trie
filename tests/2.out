m
-
-
h
x
l
-
-
-
-
-
-
-
-
w
-
-
k
f
-
c
p
-
t
-
-
m
u
y
h
x
l
r
n
o
v
a
i
z
q
w
s
g
k
f
j
c
p
e
t
d
b